nixos/doc/manual/release-notes/rl-1609.xml

```diff
@@ -39,6 +39,9 @@ following incompatible changes:</para>
     <command>start</command>, <command>stop</command>,
     <command>restart</command>, <command>status</command>.</para>
   </listitem>
+  <listitem>
+    <para><literal>hardware.pulseaudio.configFile</literal> must now be available on evaluation time, previously it could be any file.</para>
+  </listitem>
```

Comment:
If you look at the newest patches, there’s now an `extraConfig`, `configFile` is back to its original inclusion at runtime (now using pulse’s `.include` directive) and can be set to `null`.